### PR TITLE
runtime/cli: migrate 19-identity-get test

### PR DESCRIPTION
**Pull request by ianvernon**
Signed-off by: Ian Vernon <ian@cilium.io>

Related-to: #1839

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

File: test/runtime/cli.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
+// Copyright 2017 Authors of Cilium
```
Comment (eloycoto) [inline test/runtime/cli.go]: 2018

File: test/runtime/cli.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		initialized = true
+	}
+
+	BeforeEach(func() {
+		initialize()
+	})
+
+	AfterEach(func() {
+		if CurrentGinkgoTestDescription().Failed {
+			vm.ReportFailed()
```
Comment (eloycoto) [inline test/runtime/cli.go]: I would add here `cilium endpoint list` command

File: test/runtime/cli.go [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+		}
+	})
+
+	Context("Identity testing", func() {
+		It("Test labelsSHA256", func() {
+			By("Creating containers")
```
Comment (eloycoto) [inline test/runtime/cli.go]: This `By` Here doesn't make too much sense to me, there is only one test and no steps in the test, so I think that can be deleted. (Doesn't add a real value)

**Review by eloycoto — CHANGES_REQUESTED** [review]

**Comment by eloycoto** [comment]
Minor changes, if you can address it, it will be great. Thanks

File: test/runtime/cli.go (hunk 3 above)
Comment (ianvernon) [reply test/runtime/cli.go]: Done.

File: test/runtime/cli.go (hunk 2 above)
Comment (ianvernon) [reply test/runtime/cli.go]: Done.

File: test/runtime/cli.go (hunk 1 above)
Comment (ianvernon) [reply test/runtime/cli.go]: Done.

**Review by eloycoto — APPROVED** [review]

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
Failure in Cilium-Bash-Tests build is unrelated to my code changes. Will file a followup ticket. As such, am merging.

**Pull request merged**